.idea/modules.xml

```diff
@@ -2,10 +2,8 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
-      <module fileurl="file://$PROJECT_DIR$/Adamastor.iml" filepath="$PROJECT_DIR$/Adamastor.iml" />
```

Comment:
What does this change? Issues with case insensitive?

Reply:
yes